src/test/groovy/org/araneforseti/boundary/fields/BooleanFieldTest.groovy

```diff
@@ -26,18 +24,19 @@ class BooleanFieldTest {
     }
 
     @Test
-    void boolean_message_uses_default_messageName() {
-        assert scenario_messages_contains(fieldName, requiredBoolean)
-        assert scenario_messages_contains(fieldName, optionalBoolean)
+    void boolean_message_uses_default_messageConfiguration() {
```

Comment:
Needs to be clearer why this is different than the assert_messageConfiguration_is_used_for_scenarios used below and why it cannot be tested as part of that